
Attaching package: 'cowplot'

The following object is masked from 'package:ggplot2':

    ggsave

Loading required package: foreach
Loading required package: iterators
Loading required package: parallel

Attaching package: 'dplyr'

The following objects are masked from 'package:stats':

    filter, lag

The following objects are masked from 'package:base':

    intersect, setdiff, setequal, union

starting worker pid=114776 on localhost:11130 at 00:17:34.637
starting worker pid=114787 on localhost:11130 at 00:17:34.878
starting worker pid=114798 on localhost:11130 at 00:17:35.120
starting worker pid=114809 on localhost:11130 at 00:17:35.359
starting worker pid=114820 on localhost:11130 at 00:17:35.598
starting worker pid=114831 on localhost:11130 at 00:17:35.835
starting worker pid=114842 on localhost:11130 at 00:17:36.068
starting worker pid=114853 on localhost:11130 at 00:17:36.299
starting worker pid=114864 on localhost:11130 at 00:17:36.532
starting worker pid=114875 on localhost:11130 at 00:17:36.767
starting worker pid=114886 on localhost:11130 at 00:17:37.005
starting worker pid=114897 on localhost:11130 at 00:17:37.238
starting worker pid=114908 on localhost:11130 at 00:17:37.472
starting worker pid=114919 on localhost:11130 at 00:17:37.707
starting worker pid=114930 on localhost:11130 at 00:17:37.941
starting worker pid=114942 on localhost:11130 at 00:17:38.172
starting worker pid=114953 on localhost:11130 at 00:17:38.402
starting worker pid=114964 on localhost:11130 at 00:17:38.636
starting worker pid=114975 on localhost:11130 at 00:17:38.867
starting worker pid=114986 on localhost:11130 at 00:17:39.104
starting worker pid=114997 on localhost:11130 at 00:17:39.352
starting worker pid=115008 on localhost:11130 at 00:17:39.593
starting worker pid=115019 on localhost:11130 at 00:17:39.831
starting worker pid=115030 on localhost:11130 at 00:17:40.068
starting worker pid=115041 on localhost:11130 at 00:17:40.303
starting worker pid=115052 on localhost:11130 at 00:17:40.540
starting worker pid=115063 on localhost:11130 at 00:17:40.787
starting worker pid=115106 on localhost:11130 at 00:17:41.028
starting worker pid=115117 on localhost:11130 at 00:17:41.267
starting worker pid=115128 on localhost:11130 at 00:17:41.505
[1] "warning, sym=1, snp=1: 0 sample"
Error in { : task 1 failed - "object 'diffdata' not found"
Calls: source ... eval -> eval -> condfreqstatistics -> %dox% -> <Anonymous>
In addition: Warning message:
In e$fun(obj, substitute(ex), parent.frame(), e$data) :
  already exporting variable(s): savedir, filename, data, symptoms, namesymptoms, symptomvariants, numsymptomvariants, snps, namesnps, snpvariants, numsnpvariants, namesnpvariants, logpriortheta, statsfunction, mciterations, mcdiscard
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
srun: error: compute-0-1: task 0: Exited with exit code 1
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
